README.md

```diff
@@ -1 +1,125 @@
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+> This is my first MV Project as a Micronaut. In this project, I created a branch and applied linters.
+
+## 🛠 Built With <a name="built-with"></a>
+* HTML 
+* CSS
+### Tech Stack <a name="tech-stack"></a>
+> Hello Microverse!
+> ### Key Features <a name="key-features"></a>
+
+- **Using linter**
+- **Using gitHub flow**
+- **Creating a branch**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Here is my first project as a Micronaut to understand setting up linter.
+
+### Prerequisites
+
+In order to run this project you need: to download it to your desired PC.
+
+### Setup
+
+Clone this repository to your desired folder:
+1. Click Code button (shown in green color)
+2. Copy URL.
+3. Please clone it with the following command: git clone git@github.com:myaccount/my-project.git
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Ye Min Thu**
+
+- GitHub: [@githubhandle](https://github.com/mryeminthu)
+- Twitter: [@twitterhandle](https://twitter.com/mryeminthu)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ye-min-thu-76456a214/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Sidebar**
+- [ ] **Footer**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, follow my Facebook Page.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank my coding partners. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **Can I use Sublime Text for this project?**
+
+  - [Any code editor can be used.]
+
+- **Is Node.js needed?**
+
+  - [Yes.]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
```

Comment:
- [ ] This section serves to give you additional information wen filling in the readme. Kindly always delete it when you are done editing this document.
![scrn](https://github.com/mryeminthu/Hello-MV-Project/assets/99656687/68a135f4-8662-4670-9b71-5ece2e7d4199)